TMS_Tests/Tests/TestRailTests.cs

```diff
@@ -3,22 +3,33 @@
     [TestFixture]
     public class TestRailTests : BaseTest
     {
+        public string projectName;
+
         [SetUp]
         public void SetUp()
         {
-            TRLoginPage.OpenTRPageByUrl();
             TRLoginPage.SuccessfulLogin();
         }
 
         [Test]
         public void CreateTestProject()
         {
-            TRDashboardPage.AddProjectButton.Click();
-            string projectName = "AKaliasinskiTestProject " + DateTime.Now;
-            TRAddProjectPage.NameField.SendKeys(projectName);
-            TRAddProjectPage.AddProjectSubmitButton.Click();
+            TRDashboardPage.AddProjectButton().Click();
+            projectName = "AKaliasinskiTestProject " + DateTime.Now;
+            TRAddProjectPage.NameField().SendKeys(projectName);
+            TRAddProjectPage.AddProjectSubmitButton().Click();
             Assert.That(TRProjectsPage.GetNewCreatedProjectElement(projectName).Displayed);
         }
+
+        [Test]
+        public void DeleteTestProject()
+        {
+            CreateTestProject();
+            TRProjectsPage.GetDeleteButtonForProject(projectName).Click();
+            TRProjectsPage.DeleteCheckbox().Click();
+            TRProjectsPage.DeleteDialogOKButtton().Click();
+            Assert.That(TRProjectsPage.ProjectIsDeleted(projectName), Is.True);
+        }
```

Comment:
your test depends on other test, not a good practice

Reply:
Updated